app/src/main/java/com/duckduckgo/app/cta/ui/CtaViewModel.kt

```diff
@@ -192,6 +193,14 @@ class CtaViewModel @Inject constructor(
         }
     }
 
+    private fun hasTrackersInformation(events: List<TrackingEvent>): Boolean =
+        events.asSequence()
```

Comment:
Just a small optimization if it's possible. Since this method is just checking the existence of at least one tracker with a displayable name, do we need to apply `distinct`? What about this instead?

```
        events.asSequence()
            .filter { it.entity?.isMajor == true }
            .map { it.entity?.displayName }
            .filterNotNull()
            .any()
```

Reply:
Done!